dataminer/Troubleshooting/Known_issues/KI_Trend_flatline_different_timezones.md

```diff
@@ -0,0 +1,33 @@
+﻿---
+uid: KI_Trend_flatline_different_timezones
+---
+
+# Trend graph flatline when client and DMA use different timezones
+
+## Affected versions
+
+This issue occurs in DataMiner 10.5.8 and later.
+
+## Cause
+
+The method for loading trend data has changed.  
+- **Before 10.5.8**: When requesting periods like *Last day* or *Last week*, the system always requested all data up to the current time.  
+- **From 10.5.8 onwards**: To reduce the amount of data requested, only additional data is requested (e.g. *Last week* no longer re-requests the *Last day* data if it has already been retrieved).  
+
+The issue arises because:  
+- Requests for *Last day* use the **client's timezone**.  
+- Requests for longer ranges (e.g. *Last week*) use the **DataMiner Agent's timezone**, and unlike before, they no longer include the data from shorter ranges (e.g. *Last day*) that have already been retrieved.
+
+When these timezones differ, part of the data is skipped, resulting in a flatline on the graph.
+
+## Workaround
+
+Set the client's timezone to match the DataMiner Agent's timezone.
+
+## Fix
+
+No fix is available yet.
```

Comment:
Has a release note already been made for the fix? If yes, it would be good to add the RN number in comment here, as it makes it easier for us to trace when a fix has become available for a known issue.